core/kazoo_documents-1.0.0/src/kz_account.erl

```diff
@@ -52,17 +51,51 @@
 -type doc() :: wh_json:object().
 -export_type([doc/0]).
 
+%%--------------------------------------------------------------------
```

Comment:
If you aren't going to actually put docs in, why include these header comments? They serve no purpose otherwise (except to clutter the code), and any editor worth its salt should allow you to jump to a function or jump between functions easily, so I don't think they're needed visually.